index.html

```diff
@@ -4,9 +4,17 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Document</title>
+    <title>Party Hard</title>
```

Comment:
Pojmenoval bych to jinak. `Czechitas`?

Reply:
prehodil jsem to a dal do https://github.com/lmc-eu/workshop-czechitas/pull/2